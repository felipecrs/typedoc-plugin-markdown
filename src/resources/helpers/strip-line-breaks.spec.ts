import * as Handlebars from 'handlebars';

import { setup } from '../../../test/utils';

describe(`[helpers] stripLineBreaks`, () => {
  beforeAll(() => {
    setup();
  });

  test(`should compile`, () => {
    const result = Handlebars.helpers.stripLineBreaks.call('line 1\n line2\n');
    expect(result).toMatchSnapshot('line 1  line2');
  });
});
