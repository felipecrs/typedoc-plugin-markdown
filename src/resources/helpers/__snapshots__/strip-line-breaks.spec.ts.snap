// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[helpers] stripLineBreaks should compile: line 1  line2 1`] = `"line 1  line2 "`;
